metals/src/main/scala/scala/meta/internal/metals/ImportedBuild.scala

```diff
@@ -68,9 +75,37 @@ object ImportedBuild {
         new DependencySourcesParams(ids)
       )
     } yield {
+      // waiting on Bloop buildTarget/javacOptions support
```

Comment:
I can take a look at that at some point and work towards merging it, this would probably be a better way.